roles/kubernetes-apps/csi_driver/gcp_pd/templates/gcp-pd-csi-controller.yml.j2

```diff
@@ -1,11 +1,10 @@
-kind: StatefulSet
+kind: Deployment
 apiVersion: apps/v1
 metadata:
   name: csi-gce-pd-controller
   namespace: kube-system
 spec:
-  serviceName: "csi-gce-pd"
-  replicas: {{ gcp_pd_csi_controller_replicas }}
+  replicas: 1
```

Comment:
Ah wait actually you should retain the previous line `replicas: {{ gcp_pd_csi_controller_replicas }}` here

Reply:
eagle eye, 

yeah, I changed it for testing but forget to revert it back after testing. UPDATING !!!

Reply:
Done